kratos/python/add_variable_utils_to_python.cpp

```diff
@@ -218,29 +224,29 @@ void AddVariableUtilsToPython(pybind11::module &m)
         .def("CopyModelPartElementalVar", &VariableUtils::CopyModelPartElementalVar<Variable<Quaternion<double>>>)
         .def("CopyModelPartElementalVar", &VariableUtils::CopyModelPartElementalVar<Variable<Vector>>)
         .def("CopyModelPartElementalVar", &VariableUtils::CopyModelPartElementalVar<Variable<Matrix>>)
-        .def("SetVectorVar", VariableUtilsSetVariable<array_1d<double,3>>)
-        .def("SetVectorVar", VariableUtilsSetVariableForFlag<array_1d<double, 3>>)
-        .def("SetScalarVar", VariableUtilsSetVariable<double>)
-        .def("SetScalarVar", VariableUtilsSetVariableForFlag<double>)
+        .def("SetVectorVar", VariableUtilsSetVariable<array_1d<double,3>>, py::arg("variable"), py::arg("value"), py::arg("nodes"), py::arg("buffer_step")=0)
```

Comment:
i am not convinced on the use of py::arg, as from my understanding it uses under the hood kwargs.

so
1 - it has a performance penalty
2 - it is not compatible with current usage

Reply:
I did it at that moment as @philbucher, it can be replaced with the lambdas (or updating to C++14 :P )

Reply:
this has nothing to do with lambdas, @RiccardoRossi s concern is with the `py::args` being problematic (args vs kwargs) (I also don't yet know abt this :D)